src/components/calcite-alert/calcite-alert.scss

```diff
@@ -1,3 +1,13 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-alert-width: sets alert width
+ * @prop --calcite-alert-spacing-token-small: sets padding above and below alert content
```

Comment:
I think these two spacing token variables should be internal / convert to just using raw values in css